Brightness Sync/AppDelegate.swift

```diff
@@ -4,79 +4,98 @@ import os
 class AppDelegate: NSObject, NSApplicationDelegate {
     let statusItem = NSStatusBar.system.statusItem(withLength: NSStatusItem.squareLength)
     let statusIndicator = NSMenuItem(title: "Starting", action: nil, keyEquivalent: "")
-    
+
+    let updateInterval = 0.1
     var syncTimer: Timer?
-    
+
     var lastBrightness: Double?
-    
+
+    let brightnessOffsetKey = "BSBrightnessOffset"
+    var brightnessOffset: Double {
+        get {
+            UserDefaults.standard.double(forKey: brightnessOffsetKey)
+        }
+        set (newValue) {
+            UserDefaults.standard.set(newValue, forKey: brightnessOffsetKey)
+        }
+    }
+
     // CoreDisplay_Display_GetUserBrightness reports 1.0 for builtin display just before applicationDidChangeScreenParameters when closing lid.
     // This is a workaround to restore the last sane value after syncing stops.
     var lastSaneBrightness: Double?
     var lastSaneBrightnessDelayTimer: Timer?
-    
+
     static let maxDisplays: UInt32 = 8
-    
+
     func applicationDidFinishLaunching(_ aNotification: Notification) {
         if let button = statusItem.button {
             button.image = #imageLiteral(resourceName: "StatusBarButtonImage")
         }
-        
+
         let menu = NSMenu()
         menu.addItem(statusIndicator)
         menu.addItem(NSMenuItem.separator())
-        
+
+        menu.addItem(NSMenuItem(title: "Brightness Offset", action: nil, keyEquivalent: ""))
```

Comment:
Open for suggestions for a better name for this. 🤷‍♂

Reply:
Hmm, can't think of any. I also wish I had a better name for the app than “Brightness Sync”...